controllers/productController.js

```diff
@@ -0,0 +1,154 @@
+const Product = require("../models/Product");
+
+const getAllProducts = async (req, res) => {  
+
+        const products = await Product.find({ createdBy: req.user._id }).sort("createdAt");
```

Comment:
Same here - you retrieve products down below so shouldn't need to also do so here unless you plan to do something different with it